src/index.html

```diff
@@ -12,16 +12,16 @@
     <h1 class="title">Leader Board</h1>
     <div class="leaderBoard">
         <div>
-            <span>Recent scores</span3>
-                <button>Refresh</button>
+            <span>Recent scores</span>
+                <button type="button" id="refreshButton">Refresh</button>
                 <div class="leaderBoardList" id="leaderBoardList"></div>
         </div>
         <div>
             <form action="#" class="addScoreForm">
                 <h2>Add your score</h2>
-                <input type="text" name="name" id="name" placeholder="enter your name"><br>
-                <input type="text" name="score" id="score" placeholder="enter your score"><br>
-                <button type="button">Submit</button>
+                <input type="text" name="name" id="name" placeholder="enter your name" required autocomplete="off"><br>
+                <input type="text" name="score" id="score" placeholder="enter your score" required autocomplete="off"><br>
+                <button type="button" id="addScoreButton">Submit</button>
             </form>
```

Comment:
- [Optional] Dear @ghreza-crypto, Though your functionality is working, It would be wise to make the `score` input field a type `number` this prevents entering inappropriate data. 
- [Optional] also, Please notice the `required` keyword works alongside the form submission. This form only gets submitted if you have a button with `type=submit`, as you can see you have a button with type=button`, and this makes your required keyword useless. Please make the button type submit.